pkg/api/pod/util_test.go

```diff
@@ -1187,21 +1187,16 @@ func TestDropProbeGracePeriod(t *testing.T) {
 		},
 	}
 
-	//enabled := true
 	for _, enabled := range []bool{true, false} {
 		for _, oldPodInfo := range podInfo {
 			for _, newPodInfo := range podInfo {
-
-				oldPodHasGracePeriod, oldPod := newPodInfo.hasGracePeriod, oldPodInfo.pod()
-
+				oldPodHasGracePeriod, oldPod := oldPodInfo.hasGracePeriod, oldPodInfo.pod()
 				newPodHasGracePeriod, newPod := newPodInfo.hasGracePeriod, newPodInfo.pod()
-
 				if newPod == nil {
 					continue
 				}
 
 				t.Run(fmt.Sprintf("feature enabled=%v, old pod %v, new pod %v", enabled, oldPodInfo.description, newPodInfo.description), func(t *testing.T) {
-
 					defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.ProbeTerminationGracePeriod, enabled)()
```

Comment:
left this line and the `for` loop for enabling/disabling the feature (discarded all the other changes I had made previously). The test cases for the feature flag disabled are still failing.